docusaurus/docs/develop/path/7_e2e_and_load_tests.md

```diff
@@ -32,21 +32,33 @@ description: End-to-End & Load Tests for PATH
 **Run E2E tests with Docker containers:**
 
 ```bash
-# Shannon E2E tests (local Anvil)
-make test_e2e_evm_shannon
+# Shannon E2E tests with default service IDs
```

Comment:
#PUC what "default service IDs` means

Reply:
I've removed the defaults idea.

Now tests can run in one of two ways:
1. `make load_test_all` - runs the load test for ALL services (for Shannon)
2. `make load_test eth` - runs the load test for `eth` only (or whatever service IDs are provided, comma separated)

I'll resolve all comments related to the "defaults" as they no longer exist.